actors/builtin/miner/miner_actor.go

```diff
@@ -165,16 +166,14 @@ func (a Actor) SubmitFallbackPoSt(rt Runtime, params *abi.OnChainPoStVerifyInfo)
 			ProvingPeriodStart:     st.PoStState.ProvingPeriodStart + ProvingPeriod,
 			NumConsecutiveFailures: 0,
 		}
+
+		st.ProvingSet = st.Sectors
+
 		return nil
 	})
 
-	_, code := rt.Send( // TODO: what does this actually do?
-		builtin.StoragePowerActorAddr,
-		builtin.MethodsPower.OnMinerFallbackPoStSuccess,
-		&adt.EmptyValue{},
-		abi.NewTokenAmount(0),
-	)
-	builtin.RequireSuccess(rt, code, "failed to notify storage power actor")
+	// TODO: register a callback with cron to check that we submitted our proof at the right time
```

Comment:
I can take in a follow up also addressing https://github.com/filecoin-project/specs-actors/issues/83

Reply:
Fwiw i am not sure this works, specifically this method is called on FallbackPoSt submission by the miner. This is arguably not when the cron callback should be registered. It should be registered on a miner's first Sector Commitment and registered with the PowerActor, that way the PowerActor has a map from miner to epoch by which a recent fallback Post must have been submitteed, and if not, it penalizes the miner.

On FallbackPoSt submission, that epoch number is incremented by `provingPeriod`.

What do you think?